prov/shm/src/smr_ep.c

```diff
@@ -561,6 +567,11 @@ void smr_format_sar(struct smr_cmd *cmd, enum fi_hmem_iface iface, uint64_t devi
 	if (cmd->msg.hdr.op != ofi_op_read_req)
 		smr_copy_to_sar(sar_msg, resp, cmd, iface, device ,iov, count,
 				&pending->bytes_done, &pending->next);
+
+	peer_smr->sar_cnt--;
+	smr_peer_data(smr)[id].sar_status = 1;
```

Comment:
I know you just copied this code but could you change this to SMR_SAR_READY for consistency, please?